src/test/java/org/gridsuite/ds/server/controller/DynamicSimulationControllerIEEE14Test.java

```diff
@@ -0,0 +1,169 @@
+                new Date(),
+                true,
+                parametersFile);
+        given(dynamicMappingClient.createFromMapping(DynamicMappingClientTest.MAPPING_NAME_01)).willReturn(Mono.just(scriptObj));
+    }
+
+    @Override
+    protected void initTimeSeriesServiceMock() throws IOException {
+        Mockito.doAnswer(new Answer<Mono<TimeSeriesGroupInfos>>() {
+            @Override
+            public Mono<TimeSeriesGroupInfos> answer(final InvocationOnMock invocation) {
+                final Object[] args = invocation.getArguments();
+                List<TimeSeries> data = (List<TimeSeries>) args[0];
+                UUID seriesUuid = null;
+                if (!data.isEmpty()) {
+                    switch (data.get(0).getMetadata().getDataType()) {
+                        case STRING:
+                            seriesUuid = UUID.fromString(TimeSeriesClientTest.TIME_LINE_UUID);
+                            break;
+                        default:
+                            seriesUuid = UUID.fromString(TimeSeriesClientTest.TIME_SERIES_UUID);
+                            break;
+                    }
+                    TIME_SERIES_MOCK_BD.put(seriesUuid, (List<TimeSeries>) args[0]);
+                }
+                return Mono.just(new TimeSeriesGroupInfos(seriesUuid));
+            }
+        }).when(timeSeriesClient).sendTimeSeries(any());
+    }
+
+    @Test
+    public void test01() throws IOException {
+        String testBaseDir = MAPPING_NAME_01;
+
+        //run the dynamic simulation (on a specific variant with variantId=" + VARIANT_1_ID + ")
+        EntityExchangeResult<UUID> entityExchangeResult = webTestClient.post()
+                .uri("/v1/networks/{networkUuid}/run?&startTime=0&stopTime=50" + "&mappingName=" + MAPPING_NAME_01, NETWORK_UUID_STRING)
+                .exchange()
+                .expectStatus().isOk()
+                .expectBody(UUID.class)
+                .returnResult();
+
+        UUID runUuid = UUID.fromString(entityExchangeResult.getResponseBody().toString());
+
+        Message<byte[]> messageSwitch = output.receive(1000 * 5, "ds.result.destination");
+        assertEquals(runUuid, UUID.fromString(messageSwitch.getHeaders().get(DynamicSimulationResultContext.RESULT_UUID).toString()));
+
+        // prepare expected result to compare
+        DynamicSimulationResult expectedResult = DynamicSimulationResultDeserializer.read(getClass().getResourceAsStream(Paths.get(DATA_IEEE14_BASE_DIR, testBaseDir, OUTPUT, RESULT_JSON).toString()));
+        String jsonExpectedTimeSeries = TimeSeries.toJson(new ArrayList(expectedResult.getCurves().values()));
+
+        // get timeseries from mock timeseries db
+        UUID timeSeriesUuid = UUID.fromString(TimeSeriesClientTest.TIME_SERIES_UUID);
+        String jsonResultTImeSeries = TimeSeries.toJson(TIME_SERIES_MOCK_BD.remove(timeSeriesUuid));
+
+        // compare result only timeseries
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
Inject Spring one instead of creating a new one?

Reply:
Done